Offline log viewer improvements

mmaslankaprv opened the pull request:
Refactored `OfflineLogViewer` to make it easier to introduce future modifications. 

<!--
See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md#pull-request-body
for more details and examples of what is expected in a PR body.

Content in this top section is REQUIRED. Describe, in plain language, the motivation
behind the change (bug fix, feature, improvement) in this PR and how the included
commits address it.

Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...

If this PR is a backport, link to the original with `Backport of PR`, e.g.
  Backport of PR #PR-NUMBER
-->

## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [x] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [ ] v25.1.x
- [ ] v24.3.x
- [ ] v24.2.x
- [ ] v24.1.x

## Release Notes

<!--
If the changes in this PR do not need to be mentioned in the release
notes, then don't add a sub-section and simply list `none`, e.g.

* none

Otherwise, adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.
If this is a backport PR, adding contents to this section will override
the release notes section inherited from the original PR to dev.

Add one or more of the sub-sections with a short description bullet
point of the change, e.g.

### Bug Fixes

* Short description of the bug fix if this is a PR to `dev` branch.

### Features

* Short description of the feature. Explain how to configure.

### Improvements

* Short description of how this PR improves existing behavior.

-->
- none

mmaslankaprv committed:
offline_log_viewer: improve the way short read is logged

Short read warning level message may indicate that something is
incorrect about the data in the log processed by the viewer. Added a
message about truncation point to reason about the short read cause.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64847-->
<!--490b4849-2da5-4637-94f3-159f88af9ad7-->
<!--title-->
## Retry command for [Build#64847](https://buildkite.com/redpanda/redpanda/builds/64847)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/e2e_shadow_indexing_test.py::ShadowIndexingInfiniteRetentionTest.test_segments_not_deleted@{"cloud_storage_type":1}
tests/rptest/tests/datalake/topic_properties_test.py::TopicPropertiesTest.test_iceberg_topic_as_read_replica_is_rejected
tests/rptest/tests/retention_policy_test.py::ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention_application@{"cloud_storage_type":2}
tests/rptest/tests/retention_policy_test.py::ShadowIndexingCloudRetentionTest.test_topic_recovery_retention_settings
tests/rptest/tests/archive_retention_test.py::CloudArchiveRetentionTest.test_delete@{"cloud_storage_type":1,"retention_type":"retention.bytes"}
tests/rptest/tests/datalake/cluster_restore_test.py::DatalakeClusterRestoreTest.test_basic@{"catalog_type":"rest_jdbc","cloud_storage_type":1}
tests/rptest/tests/e2e_shadow_indexing_test.py::EndToEndShadowIndexingTest.test_reset_from_cloud@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast1@{"cloud_storage_type":2}
tests/rptest/tests/timequery_test.py::TimeQueryTest.test_timequery@{"batch_cache":false,"cloud_storage":true,"spillover":true}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast3@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_missing_partition@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_time_based_retention@{"cloud_storage_type":2}
tests/rptest/tests/archival_test.py::ArchivalTest.test_timeboxed_uploads@{"acks":0,"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_prevent_recovery@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"no_check","cloud_storage_type":2}
tests/rptest/tests/delete_records_test.py::DeleteRecordsTest.test_delete_records_segment_deletion@{"cloud_storage_enabled":true,"truncate_point":"at_high_watermark"}
tests/rptest/tests/delete_records_test.py::DeleteRecordsTest.test_delete_records_segment_deletion@{"cloud_storage_enabled":true,"truncate_point":"start_offset"}
```
<!--/body-->

vbotbuildovich commented:
<!--TEST RESULTS-->
<!--b1cf1f48-2390-4bf3-85fe-0e32795d256c-->
<!--TEST RESULTS-->
<!--593c6f52-9bde-49f5-98e6-9468f1072dca-->
<!--TEST RESULTS-->
<!--d59d5fe5-b71f-4962-acb7-9087f84d315a-->
<!--TEST RESULTS-->
<!--f8c59b2f-0a86-466b-9370-0760d05e789b-->
<!--title-->
## CI test results
<!--/title-->
<!--body-->
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64847">build#64847</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.alter_topic_configuration_test.ShadowIndexingGlobalConfig.test_topic_manifest_reupload | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/2 |
| rptest.tests.alter_topic_configuration_test.ShadowIndexingGlobalConfig.test_topic_manifest_reupload | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=-1.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=-1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/2 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=-1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=0.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=0.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/5 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=0.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=0.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=1.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.archival_test.ArchivalTest.test_timeboxed_uploads.acks=1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.ABS.retention_type=retention.bytes | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.ABS.retention_type=retention.bytes | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.ABS.retention_type=retention.ms | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.ABS.retention_type=retention.ms | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.S3.retention_type=retention.bytes | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.S3.retention_type=retention.bytes | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.archive_retention_test.CloudArchiveRetentionTest.test_delete.cloud_storage_type=CloudStorageType.S3.retention_type=retention.ms | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=20.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=20.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_cloud_retention.max_consume_rate_mb=None.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_gc_entire_manifest.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.cloud_retention_test.CloudRetentionTest.test_gc_entire_manifest.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.cloud_retention_test.CloudRetentionTimelyGCTest.test_retention_with_node_failures.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.cloud_storage_scrubber_test.CloudStorageScrubberTest.test_scrubber.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.cloud_storage_scrubber_test.CloudStorageScrubberTest.test_scrubber.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.cloud_storage_scrubber_test.CloudStorageScrubberTest.test_scrubber.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage.cleanup_policy=compact.delete | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage.cleanup_policy=delete | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves.cleanup_policy=delete | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.cloud_storage_usage_test.CloudStorageUsageTest.test_cloud_storage_usage_reporting | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.cloud_storage_usage_test.CloudStorageUsageTest.test_cloud_storage_usage_reporting | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.cloud_storage_usage_test.CloudStorageUsageTest.test_cloud_storage_usage_reporting_with_partition_moves | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.cluster_config_test.ClusterConfigAzureSharedKey.test_live_shared_key_change.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.cluster_config_test.ClusterConfigAzureSharedKey.test_live_shared_key_change.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/6 |
| rptest.tests.cluster_recovery_test.ClusterRecoveryTest.test_basic_controller_snapshot_restore | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.cluster_recovery_test.ClusterRecoveryTest.test_basic_controller_snapshot_restore | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.cluster_recovery_test.ClusterRecoveryTest.test_bootstrap_with_recovery | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.cluster_recovery_test.ClusterRecoveryTest.test_bootstrap_with_recovery | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.consumer_group_recovery_test.ConsumerOffsetsRecoveryTest.test_consumer_offsets_partition_recovery | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.consumer_group_recovery_test.ConsumerOffsetsRecoveryTest.test_consumer_offsets_partition_recovery | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.NESSIE | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.NESSIE | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/21 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_HADOOP | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_HADOOP | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_JDBC | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.datalake.cluster_restore_test.DatalakeClusterRestoreTest.test_basic.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_JDBC | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.datalake.topic_properties_test.TopicPropertiesTest.test_iceberg_topic_as_read_replica_is_rejected | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.datalake.topic_properties_test.TopicPropertiesTest.test_iceberg_topic_as_read_replica_is_rejected | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=at_high_watermark | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=at_high_watermark | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=at_segment_boundary | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=at_segment_boundary | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=one_below_high_watermark | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=random_offset | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=random_offset | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/5 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=start_offset | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.delete_records_test.DeleteRecordsTest.test_delete_records_segment_deletion.cloud_storage_enabled=True.truncate_point=start_offset | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_recover_after_delete_records | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_from_cloud.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_from_cloud.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/7 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_from_cloud.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_spillover.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_spillover.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_spillover.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTest.test_reset_spillover.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTestCompactedTopic.test_compacting_during_leadership_transfer.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTestCompactedTopic.test_compacting_during_leadership_transfer.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndShadowIndexingTestCompactedTopic.test_write.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndThrottlingTest.test_throttling.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.EndToEndThrottlingTest.test_throttling.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/9 |
| rptest.tests.e2e_shadow_indexing_test.ShadowIndexingInfiniteRetentionTest.test_segments_not_deleted.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_shadow_indexing_test.ShadowIndexingInfiniteRetentionTest.test_segments_not_deleted.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.e2e_shadow_indexing_test.ShadowIndexingInfiniteRetentionTest.test_segments_not_deleted.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore.message_size=5000.num_messages=100000.recovery_overrides=.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore.message_size=5000.num_messages=100000.recovery_overrides=.retention.local.target.bytes.1024.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore.message_size=5000.num_messages=100000.recovery_overrides=.retention.local.target.bytes.1024.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_config_batches.num_messages=2.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_config_batches.num_messages=2.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_config_batches.num_messages=2.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.remote_label_test.RemoteLabelsTest.test_clusters_share_bucket.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.remote_label_test.RemoteLabelsTest.test_clusters_share_bucket.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.remote_label_test.RemoteLabelsTest.test_clusters_share_bucket.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/6 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention_application.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention_application.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_size_based_retention_application.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_time_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_time_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_cloud_time_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/5 |
| rptest.tests.retention_policy_test.ShadowIndexingCloudRetentionTest.test_topic_recovery_retention_settings | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.services_self_test.BucketScrubSelfTest.test_missing_segment.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/8 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/9 |
| rptest.tests.shadow_indexing_compacted_topic_test.TSWithAlreadyCompactedTopic.test_initial_upload | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.timequery_test.TimeQueryTest.test_timequery.cloud_storage=True.batch_cache=False.spillover=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/7 |
| rptest.tests.timequery_test.TimeQueryTest.test_timequery.cloud_storage=True.batch_cache=False.spillover=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.drop_lifecycle_marker_test.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.drop_lifecycle_marker_test.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.drop_lifecycle_marker_test.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.partition_movement_test.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.partition_movement_test.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_cloud_storage_test.disable_delete=False.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_cloud_storage_test.disable_delete=True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_cloud_storage_test.disable_delete=True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_installed_snapshots_test | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_unavailable_test.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_empty_segments.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_empty_segments.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_empty_segments.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/4 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast1.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast1.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast1.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast2.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/10 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast2.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/7 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast2.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast3.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast3.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast3.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_fast3.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.ABS.check_mode=check_manifest_and_segment_metadata | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/8 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.ABS.check_mode=check_manifest_existence | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/6 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.ABS.check_mode=check_manifest_existence | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.ABS.check_mode=no_check | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.ABS.check_mode=no_check | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.S3.check_mode=check_manifest_and_segment_metadata | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.S3.check_mode=check_manifest_and_segment_metadata | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.S3.check_mode=check_manifest_existence | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/8 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.S3.check_mode=no_check | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/6 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_many_partitions.cloud_storage_type=CloudStorageType.S3.check_mode=no_check | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_partition.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_partition.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_partition.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_partition.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_segment.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/9 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_segment.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/7 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_missing_segment.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_no_data.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_no_data.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_no_data.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/5 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_prevent_recovery.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_prevent_recovery.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_prevent_recovery.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_prevent_recovery.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_size_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_size_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_size_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/4 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_time_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4f87-91b7-f29883f7b52b | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_time_based_retention.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c6-4d1e-9a8b-481caa02e4fd | FAIL | 0/21 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_time_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3461-4738-9e2a-8edeb34b1ecc | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_time_based_retention.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_vcluster_id.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4c7c-ae1f-3e764896e708 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_vcluster_id.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/1 |
| rptest.tests.topic_recovery_test.TopicRecoveryTest.test_vcluster_id.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/4 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c4-41f9-aae6-801603087367 | FAIL | 0/1 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d35-3460-4260-91e1-fe372c3e2230 | FAIL | 0/7 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64847#01965d47-55c5-4f4a-926e-545eafaf6977 | FAIL | 0/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64957">build#64957</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-----------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_storage_scrubber_test.CloudStorageScrubberTest.test_scrubber.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64957#019663d0-eb6b-4445-8f45-1f2fe73037f2 | FLAKY | 20/21 |
| rptest.tests.consumer_group_test.ConsumerGroupTest.test_basic_group_join.static_members=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64957#019663d0-eb6a-4958-9a88-9400ecc39c0d | FAIL | 0/21 |
| rptest.tests.consumer_group_test.ConsumerGroupTest.test_basic_group_join.static_members=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64957#019663d6-229b-47f7-ad9e-e6af1f199850 | FAIL | 0/21 |
| rptest.tests.consumer_group_test.ConsumerGroupTest.test_basic_group_join.static_members=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64957#019663d0-eb6b-4445-8f45-1f2fe73037f2 | FAIL | 0/21 |
| rptest.tests.consumer_group_test.ConsumerGroupTest.test_basic_group_join.static_members=True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64957#019663d6-2297-43b1-967c-b07f3beeb77c | FAIL | 0/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/65002">build#65002</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.datalake.disk_budget_test.DatalakeDiskUsageTest.test_idle_finish.num_partitions=10.concurrent_translations=4.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966783-a503-4554-bedf-356dc24f8c4c | FAIL | 0/1 |
| rptest.tests.datalake.schema_scale_test.SchemaScaleTest.schema_scale_test.cloud_storage_type=CloudStorageType.S3.query_engine=QueryEngineType.SPARK.catalog_type=CatalogType.NESSIE.use_partition_spec=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966768-5279-446c-8774-8302ad1e6879 | FLAKY | 20/21 |
| rptest.tests.partition_balancer_test.PartitionBalancerTest.test_unavailable_nodes | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966783-a502-4a4f-b58a-bccd926ad1f1 | FLAKY | 20/21 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966783-a502-4a4f-b58a-bccd926ad1f1 | FLAKY | 14/21 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966768-5277-480f-8840-d6dac4043483 | FLAKY | 17/21 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/65002#01966783-a503-4554-bedf-356dc24f8c4c | FLAKY | 14/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/65092">build#65092</a></summary>


| test_id                                                                                                                                          | test_kind   | job_url                                                                                   | test_status   | passed   |
|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves.cleanup_policy=compact.delete | ducktape    | https://buildkite.com/redpanda/redpanda/builds/65092#01966c2f-e1c9-4d9c-8682-130dd50bc8ce | FLAKY         | 18/21    |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False                                             | ducktape    | https://buildkite.com/redpanda/redpanda/builds/65092#01966c2f-e1cc-47d4-bb5e-dc884024c699 | FAIL          | 0/1      |
</details>
<!--/body-->

mmaslankaprv committed:
offline_log_viewer: added missing batch types

Signed-off-by: Michał Maślanka <michal@redpanda.com>

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64957-->
<!--7153983d-7b91-46e4-8d69-38b9c1edf166-->
<!--57df9d67-713e-49f9-9bb9-cc3411f610c7-->
<!--55e691bc-c418-4751-9445-12dbfd389c84-->
<!--8c1821d9-9e25-4173-bc65-c70fdbbc3516-->
<!--title-->
## Retry command for [Build#64957](https://buildkite.com/redpanda/redpanda/builds/64957)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/consumer_group_test.py::ConsumerGroupTest.test_basic_group_join@{"static_members":false}
tests/rptest/tests/consumer_group_test.py::ConsumerGroupTest.test_basic_group_join@{"static_members":true}
tests/rptest/tests/cloud_storage_scrubber_test.py::CloudStorageScrubberTest.test_scrubber@{"cloud_storage_type":1}
```
<!--/body-->

bharathv commented:
failure seems related.

mmaslankaprv committed:
offline_log_viewer: moved offline log viewer to a class

Moved viewer implementation to a class to make it easier to modify it

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
offline_log_viewer: removed force option

Removed validation checking if controller log directory exists. The
validation didn't make any sense and we always had to pass the `--force`
flag anyway.

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
offline_log_viewer: added option to print only one partition

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
offline_log_viewer: added ability to print all consumer offset batches

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
offline_log_viewer: include term in batch object

Signed-off-by: Michał Maślanka <michal@redpanda.com>

mmaslankaprv committed:
offline_log_viwer: consumer offsets summary

Signed-off-by: Michał Maślanka <michal@redpanda.com>

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/65002-->
<!--78dd5616-2801-4bdf-880f-fa95548727f7-->
<!--65ed39e4-d267-410a-b46a-b60d3a89ec17-->
<!--1a36c758-c52c-4529-aa5c-308a22271743-->
<!--title-->
## Retry command for [Build#65002](https://buildkite.com/redpanda/redpanda/builds/65002)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/datalake/schema_scale_test.py::SchemaScaleTest.schema_scale_test@{"catalog_type":"nessie","cloud_storage_type":1,"query_engine":"spark","use_partition_spec":false}
tests/rptest/tests/partition_balancer_test.py::PartitionBalancerTest.test_unavailable_nodes
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/datalake/disk_budget_test.py::DatalakeDiskUsageTest.test_idle_finish@{"cloud_storage_type":1,"concurrent_translations":4,"num_partitions":10}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
```
<!--/body-->

andrwng commented on tools/offline_log_viewer/storage.py:
```diff
@@ -289,8 +293,14 @@ class Segment:
         self.path = path
 
+    def _term(self):
+        m = SEGMENT_NAME_PATTERN.match(os.path.basename(self.path))
```
nit: maybe do this in the constructor and cache the term? It does't change, does it? Though I guess we'll create the iterator once anyway

mmaslankaprv replied:
done

secpanda commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/devprod/pr-checks/ef8eed41-ee0d-4f02-af3b-fe53a476dfa3?refs=df6ac25910ceb62b6ed7f6bc18713b41a8a2f6ee&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/devprod/pr-checks/ef8eed41-ee0d-4f02-af3b-fe53a476dfa3/license?refs=df6ac25910ceb62b6ed7f6bc18713b41a8a2f6ee&source=prChecksComment)

secpanda commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/devex-c3w/pr-checks/7487beb0-8543-4dc3-9962-d96d2519df1c?refs=df6ac25910ceb62b6ed7f6bc18713b41a8a2f6ee&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/devex-c3w/pr-checks/7487beb0-8543-4dc3-9962-d96d2519df1c/license?refs=df6ac25910ceb62b6ed7f6bc18713b41a8a2f6ee&source=prChecksComment)

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/65092-->
<!--af483e96-4f86-4888-9303-75f257b9bcc5-->
<!--title-->
## Retry command for [Build#65092](https://buildkite.com/redpanda/redpanda/builds/65092)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/cloud_storage_timing_stress_test.py::CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves@{"cleanup_policy":"compact,delete"}
```
<!--/body-->
